cmd/proxy/actions/cache_miss_handler.go

```diff
@@ -6,11 +6,12 @@ import (
 
 	"github.com/gobuffalo/buffalo"
 	"github.com/gobuffalo/buffalo/worker"
+	"github.com/gomods/athens/pkg/modfilter"
 	"github.com/gomods/athens/pkg/paths"
 	"github.com/gomods/athens/pkg/storage"
 )
 
-func cacheMissHandler(next buffalo.Handler, w worker.Worker) buffalo.Handler {
+func cacheMissHandler(next buffalo.Handler, w worker.Worker, mf *modfilter.ModFilter) buffalo.Handler {
 	return func(c buffalo.Context) error {
 		nextErr := next(c)
```

Comment:
wouldn't we want to check if the module should not be processed before we go to the next step?